### PR TITLE
Tidy lib directory

### PR DESCRIPTION
**Pull request by threedaymonk**
The lib directory is a disorganised junk drawer. This moves the files found there to more appropriate locations.

### CODE REVIEW

File: spec/services/email_checker_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
-require 'email_validator'
+require 'email_checker'
```
Comment (dan-palmer) [inline spec/services/email_checker_spec.rb]: I think this `require` is redundant

File: lib/tasks/static_pages.rake [hunk 2]
```diff
@@ -20,7 +20,7 @@ namespace :static_pages do
       outpath = File.join([Rails.root, 'public', output])
       resp = app.get(route)
       if resp == 200
-        File.delete(outpath) unless not File.exist?(outpath)
+        File.delete outpath if File.exist?(outpath)
```
Comment (dan-palmer) [inline lib/tasks/static_pages.rake]: still :laughing:

File: spec/models/prison_day_spec.rb [hunk 3]
```diff
@@ -81,12 +81,16 @@
...
 
         context 'and has an anomalous booking slot' do
-          subject { described_class.new(
-            bank_holiday_friday,
-            prison_from(prison_data.merge(
-              slot_anomalies: {bank_holiday_friday => ["0930-1000"]}
-            ))
-          )}
+          subject {
+            described_class.new(
```
Comment (dan-palmer) [inline spec/models/prison_day_spec.rb]: :+1: :pray:

File: spec/services/email_checker_spec.rb (hunk 1 above)
Comment (threedaymonk) [inline spec/services/email_checker_spec.rb]: I think you're right.

File: lib/tasks/static_pages.rake (hunk 2 above)
Comment (SteveMarshall) [inline lib/tasks/static_pages.rake]: I don't not like undoing the addition of `unless not` :+1:

Comment (threedaymonk) [inline lib/tasks/static_pages.rake]: I could not fail to disagree with both of you less.

**Comment by threedaymonk** [comment]
This is now waiting on #357, which is likely to precipitate some merge conflicts.

File: lib/tasks/static_pages.rake (hunk 2 above)
Comment (tatyree) [inline lib/tasks/static_pages.rake]: All I can not succeed at deducting from this conversation is "Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo"

Comment (threedaymonk) [inline lib/tasks/static_pages.rake]: John, where Mark had had had had had had had. Had had had had a better effect on the teacher.

**Pull request merged**